src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -506,4 +506,39 @@ public function frontendDevDisable($siteDir = 'default', array $opts = ['yes|y'
             ->remove($this->devServicesPath)
             ->run();
     }
+
+    /**
+     * Setup a site in GitHub Codespaces.
+     *
+     * @throws \Robo\Exception\TaskException
+     */
+    public function setupCodespaces()
+    {
+        $codespaces_directory = getenv('PWD') . '/web';
```

Comment:
This returns the default directory at first, which is  `/workspaces/ {{reponame}}`. I'm appending `/web` here to match CHQ, but some projects use `docroot` still?

Reply:
```
$docRootDir = Robo::config()->get('drupal_document_root') ?? 'web';
```
Does this line help?

I stole it from elsewhere in the file.

Reply:
Example in the wild:

- https://github.com/ChromaticHQ/imprivata-drupal-8/blob/master/robo.yml#L26